Header header
geometry_msgs/Point32[] wallPts
uint8[] colors
uint16 keyInd
uint16 patchInd
uint16 rows
uint16 cols
